Simbody/include/simbody/internal/SimbodyMatterSubsystem.h

```diff
@@ -206,15 +206,22 @@ geometry that can be used to visualize this multibody system. **/
 bool getShowDefaultGeometry() const;
 
 /** The number of bodies includes all mobilized bodies \e including Ground,
-which is the 0th mobilized body. (Note: if special particle handling were
-implmemented, the count here would \e not include particles.) Bodies and their
-inboard mobilizers have the same index since they are grouped together as a 
-MobilizedBody. MobilizedBody numbering (using unique integer type
-MobilizedBodyIndex) starts with Ground at MobilizedBodyIndex(0) with a regular
-labeling such that children have higher indices than their parents. Ground
-does not have a mobilizer (or I suppose you could think of its mobilizer as 
-the Weld joint that attaches it to the universe), but otherwise 
-every mobilized body has a unique body and mobilizer. **/
+which is the 1st mobilized body, at the 0th MobilizedBodyIndex. (Note: if 
```

Comment:
"first" would be better here to avoid using the numeric value that might look like an index. And then "at MobilizedBodyIndex 0 for more contrast".